src/racer.c

```diff
@@ -578,6 +582,7 @@ void obj_init_racer(Object* obj, LevelObjectEntry_CharacterFlag* arg1) {
     tempObj->unk1C6 = 100;
     D_8011D580 = 0;
 
+    // This needs to be on one line to match.
     for (i = 0; i < 4; i++) { D_8011D58C[i] = 0; }
```

Comment:
Just guessing, but that `i< 4` looks like it's just looping through possible players. If so, should it be `i < MAXCONTROLLERS`?